docs.feldera.com/docs/connectors/sources/s3.md

```diff
@@ -26,14 +26,25 @@ The S3 input connector supports [fault tolerance](/pipelines/fault-tolerance).
 | `aws_access_key_id`           | string |            | AWS Access Key id. This property must be specified unless `no_sign_request` is set to `true`.|
 | `aws_secret_access_key`       | string |            | Secret Access Key. This property must be specified unless `no_sign_request` is set to `true`.|
 | `no_sign_request`             | bool   | `false`    | Do not sign requests. This is equivalent to the `--no-sign-request` flag in the AWS CLI. |
-| `key`                         | string |            | Read a single object specified by a key. Either this property or the `prefix` property must be set. |
-| `prefix`                      | string |            | Read all objects whose keys match a prefix. Set to an empty string to read all objects in the bucket. Either this property or the `key` property must be set. |
-| `region`*                     | string |            | AWS region. |
-| `bucket_name`*                | string |            | S3 bucket name. |
+| `key`[^1]                     | string |            | Read a single object specified by a key. |
+| `prefix`[^1]                  | string |            | Read all objects whose keys match a prefix. Set to an empty string to read all objects in the bucket. |
+| `region`[^2]                  | string |            | AWS region. |
+| `bucket_name`[^2]             | string |            | S3 bucket name. |
 | `endpoint_url`                | string |            | The endpoint URL used to communicate with this service. Explicitly set it to connect to non-AWS services. For example, use `https://storage.googleapis.com` to interact with Google Cloud Storage. |
 | `max_concurrent_fetches`      | integer| 8          | <p>Controls the number of S3 objects fetched in parallel.</p><p>Increasing this value can improve throughput by enabling greater concurrency. However, higher concurrency may lead to timeouts or increased memory usage due to in-memory buffering.</p><p>Recommended range: 1–10. Default: 8.</p> |
 
-*Fields marked with an asterisk are required.
+:::tip
+When `prefix` is used, Feldera reads objects in alphabetical order,
+the order returned by S3's [ListObjectsV2] operation.  When
+`max_concurrent_fetches` is greater than 1, data is processed in
+parallel for up to the specified number of objects, rather than
+sequentially from one object to the next.
+:::
+
+[^1]: Exactly one of `key` or `prefix` must be set.
+[^2]: This property is required.
```

Comment:
I think the region gets picked up by IRSA right? In which case, it doesn't need to be specified in the connector?